### DISCUSSION TITLE
Automatic feeder with summarization

### DISCUSSION THREAD

**Discussion by ShahKshitij31** [carrot2/carrot2#1477] [Q&A]
Hi, I want to build an automatic feeder, with auto-summarization from PubMed. I want the search bar on the main webpage for my website, so can you help me figure out how to use your repository in developing my website?

**Comment by dweiss** [comment]
The source code is available - please take a look at how the pubmed document source works in the demo frontend, this should give you some ideas on how to integrate PubMed API with Carrot2 or other systems that perform text summarization (there are now many). We don't have the time to guide you through more specific code sections, I'm afraid - sorry about that.
